kaggle/RANZCR/4th_place_solution/README.md

```diff
@@ -4,9 +4,16 @@ https://www.kaggle.com/c/ranzcr-clip-catheter-line-classification
 
 # Preparations
 
-Except monai required libraries, please ensure that `opencv-python` is installed. The command could be:
+Recommend to use MONAI docker:
+
+```
+docker pull projectmonai/monai:latest
+```
+
+Otherwise, e1xcept monai required libraries, please ensure that `opencv-python` and `scipy` are installed. The command could be:
```

Comment:
```suggestion
Otherwise, except for monai required libraries, please ensure that `opencv-python` and `scipy` are installed. The command could be:
```